server/src/types/attainable.ts

```diff
@@ -0,0 +1,24 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+export interface AttainableRequestData {
```

Comment:
This type is likely to just be used in a single file, so it could be defined in the attainable controller file, but it's not a big deal.

Reply:
It is also used in test mock data, I'll keep it as is for now.